Samples/iOS-Swift/iOS-Swift6UITests/UITests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+
+final class UITests: XCTestCase {
```

Comment:
`m`: As this can lead to crashes we could add this to UI tests critical and migrate this test to maestro.

Reply:
We're using Mastro for tests that require the app to crash and be reopened.
This is not the case. If the app closes in this test, it's because something went wrong.
I prefer to avoid an external tool (Maestro in this case) as much as possible.

Reply:
Also fine.